src/node/rpc/member_frontend.h

```diff
@@ -387,6 +387,25 @@ namespace ccf
         caller_id, {cose_sign1.begin(), cose_sign1.end()});
     }
 
+    bool is_proposal_replayed_or_stale(
```

Comment:
IIUC, this is currently only `is_proposal_replayed`, and the _stale_ detection is still WIP? If so, we should rename this function for now to avoid confusion.

Reply:
Yes and yes, since the header isn't exported.